src/main/java/com/iexec/worker/compute/ComputeManagerService.java

```diff
@@ -169,16 +170,27 @@ public AppComputeResponse runCompute(TaskDescription taskDescription,
         AppComputeResponse appComputeResponse =
                 appComputeService.runCompute(taskDescription, secureSessionId);
 
-        if (appComputeResponse.isSuccessful() && !appComputeResponse.getStdout().isEmpty()) {
-            // save /output/stdout.txt file
-            String stdoutFilePath =
-                    workerConfigService.getTaskIexecOutDir(chainTaskId) + File.separator + STDOUT_FILENAME;
-            File stdoutFile = FileHelper.createFileWithContent(stdoutFilePath
-                    , appComputeResponse.getStdout());
-            log.info("Saved stdout file [path:{}]",
-                    stdoutFile.getAbsolutePath());
-            //TODO Make sure stdout is properly written
-        }
+        if (appComputeResponse.isSuccessful())
+            if (!appComputeResponse.getStdout().isEmpty()) {
+                // save /output/stdout.txt file
+                String stdoutFilePath =
+                        workerConfigService.getTaskIexecOutDir(chainTaskId) + File.separator + STDOUT_FILENAME;
+                File stdoutFile = FileHelper.createFileWithContent(stdoutFilePath
+                        , appComputeResponse.getStdout());
+                log.info("Saved stdout file [path:{}]",
+                        stdoutFile.getAbsolutePath());
+                //TODO Make sure stdout is properly written
+            }
+            if (!appComputeResponse.getStderr().isEmpty()) {
```

Comment:
Single method for writing std? (something like:)
```
writeStd(appComputeResponse.getStdout(),STDOUT_FILENAME)
writeStd(appComputeResponse.getStderr(), STDERR_FILENAME)
```

Reply:
Good point, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/448/commits/8c658b4a32f4983869b8b65194d62f122bfa9ded.